### ISSUE TITLE
Sidebar form elements missing labels

### ISSUE THREAD

**Issue by smcmurtry** [veteransaffairscanada/vac-benefits-directory#2030]
Form elements must have labels 

* Tags: label
* Issue: IDs used in ARIA and labels must be unique (duplicate-id-aria:)
* Snippet: `<input type="radio" id="veteran-0" class="css-hhofyz" value="veteran">`

How to fix
Form element has multiple <label> elements
Form element has explicit <label> that is hidden

Programmatically associate labels to all form controls and ensure there are no duplicate labels. You can do so by using an implicit `labelelement` and explicit `label`, `aria-label`, or `aria-labelledby` attribute values.

Form elements that should have labels
* Text entry fields, e.g. `<input type="text">`, `<input type="password">` and `<textarea>`
* Radio buttons, `<input type="radio">`
* Checkboxes, `<input type="checkbox">`
* Select menus, `<select>`

The only exceptions for this requirement are:
* Buttons — buttons are self-labeling
* Hidden inputs — Inputs with the type attribute value of hidden (e.g., `type="hidden"`). These inputs are hidden and unavailable for user input. They therefore need no label.
